src/cascadia/TerminalSettingsModel/ActionArgs.idl

```diff
@@ -122,10 +122,22 @@ namespace Microsoft.Terminal.Settings.Model
         All = 0xffffffff,
     };
 
-    [default_interface] runtimeclass NewTerminalArgs {
+    interface INewContentArgs {
+        String Type;
+        Boolean Equals(INewContentArgs other);
+        UInt64 Hash();
+        INewContentArgs Copy();
+        String GenerateName();

```

Comment:
FWIW, I believe property-like functions are usually prefixed with `Get...`. `Type` should probably not be settable either.

Reply:
`Equals`, `Hash`, `Copy`, and `GenerateName` were all just straight lifted out of NewTerminalArgs. I can rename them if you think it's important, but also, 🤷